<a href="https://colab.research.google.com/github/imshubhamk23/Python_Assignments/blob/main/Programming_Assignment_16.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [ ]:
def stutter(word):
        res = (word[0] + word [1] + "... ") * 2 + word + "?"
        print(res)

In [ ]:
stutter("incredible")

in... in... incredible?


In [ ]:
#Angle in Radians × 180°/π = Angle in Degrees

def radians_to_degrees(angle):
    res = angle * 180/3.14159
    print(round(res,1))

In [ ]:
radians_to_degrees(1)
radians_to_degrees(20)
radians_to_degrees(50)

57.3
1145.9
2864.8


In [ ]:
def is_curzon(num):
    a = 2 ** num + 1
    b = 2 * num + 1
    if a % b == 0:
        return True
    else:
        return False

In [ ]:
is_curzon(5)

True

In [ ]:
is_curzon(10)

False

In [ ]:
is_curzon(14)

True

In [ ]:
def area_of_hexagon(s):
    area = (3 * (3 ** 0.5) * (s ** 2))/2
    print(round(area,1))

In [ ]:
area_of_hexagon(1)
area_of_hexagon(2)
area_of_hexagon(3)

2.6
10.4
23.4


Question 4.Given the side length x find the area of a hexagon.

Examples
area_of_hexagon(1) ➞ 2.6
area_of_hexagon(2) ➞ 10.4
area_of_hexagon(3) ➞ 23.4

In [ ]:
def binary(num):
    unit = 1
    binary = 0
    rem = 0
    while num > 0:
        rem = num % 2
        binary = (rem*unit)+binary
        unit = unit*10
        num = num//2
    print(binary)

In [ ]:
binary(1)
binary(5)
binary(10)

1
101
1010
